botocore/parsers.py

```diff
@@ -577,7 +590,7 @@ def _do_parse(self, response, shape):
         return self._parse_body_as_xml(response, shape, inject_metadata=True)
 
     def _parse_body_as_xml(self, response, shape, inject_metadata=True):
-        xml_contents = response['body']
+        xml_contents = response['body'] or b'<xml/>'
```

Comment:
Is there something unique about this case or should this be handled farther down in `_parse_xml_string_to_dom`? I'm curious what valid cases we're expecting the service to send us a body and we're getting nothing.